src/components/Documentation/Layout/SidebarMenu/index.tsx

```diff
@@ -8,6 +8,7 @@ import includes from 'lodash/includes'
 import ShowOnly from '../../../ShowOnly'
 import DownloadButton from '../../../DownloadButton'
 import Link from '../../../Link'
+import { ReactComponent as ExternalLinkIcon } from '../../../../../static/img/external-link-minimal.svg'
```

Comment:
does it make sense to just put it nearby? like we do with css and I think other assets already

Reply:
It looks to me like nearly all image assets, especially icons like this, live in `/static`.  
But where the new icon is only used here right now I see no problem in co-locating it with the component. It can always be moved up if we decide to use it elsewhere.

Reply:
👍 thanks!

There is an example already - src/components/Blog/Post/Share/icons

It makes sense to me, to be honest - easier to understand and keep track, and path is simpler, etc. I think, moving forward we should try to move all the images close to MD or components. We even have a ticket for this I believe. What is your take on this?